scripts/get_command_info.py

```diff
@@ -0,0 +1,210 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+import argparse
+import json
+import os
+from os.path import join
+
+"""Valkey command categorizer
```

Comment:
This is a very good work!

I am a bit confused - Dont we want to align the RouteBy::SecondArg case with its output? If I understand correctly the SecondArg commands should be the following 2 sets:
```
============================
Category: SecondArg commands
Description: Commands with their first key argument at position 2
List of commands in this category:

BITOP
MEMORY USAGE
OBJECT ENCODING
OBJECT FREQ
OBJECT IDLETIME
OBJECT REFCOUNT
PFDEBUG
XGROUP CREATE
XGROUP CREATECONSUMER
XGROUP DELCONSUMER
XGROUP DESTROY
XGROUP SETID
XINFO CONSUMERS
XINFO GROUPS
XINFO STREAM


============================
Category: SecondArgAfterKeyCount commands
Description: Commands with their first key argument at position 2, after a numkeys argument
List of commands in this category:

LMPOP
SINTERCARD
ZDIFF
ZINTER
ZINTERCARD
ZMPOP
ZUNION
```

So per the script's output - both @Yury-Fridlyand and @shohamazon suggestions on FCALL and BZMPOP are not SecondArg?

In addition, please leverage your awesome script to complete the list - it seems that some commands are missing (e.g. MEMORY USAGE)...

Please merge when done

Reply:
- FCALL is being categorized by the script into ThirdArgAfterKeyCount, the signature is 
`FCALL function numkeys [key [key ...]] [arg [arg ...]]`. The key arg is optional, so if a key is provided it will be at position 3. Is ThirdArgAfterKeyCount still the correct category even though the key arg is optional? If not, what category should it be?
- BZMPOP is being categorized by the script into ThirdArgAfterKeyCount, I think this is correct as its signature is `BZMPOP timeout numkeys key [key ...] <MIN | MAX> [COUNT count]` 
- MEMORY USAGE is in the SecondArgs category, you can see it in the output snippet you posted above. I just checked and the output number of commands matches the number of json files

There's just a couple things that I still need to complete in this PR:
- I'm not sure how to distinguish between commands that are mapped to RouteBy::Random vs RouteBy::Undefined based on their json. For example, AUTH is mapped to Random, how should the script determine this based on its json info ([here](https://github.com/valkey-io/valkey/blob/26388270f197bce84817eea0a73d687d58442654/src/commands/auth.json))?
- Once I add logic to map to the random category after resolving the above question, I'll have to add the outputted commands to their categories in cluster_routing.rs

Reply:
For `FCALL` we need another category, I think. In scope of current PR we can leave a TODO note only.
@barshaul what do you think?

Reply:
Bar answered my question about the random category on slack, she said to leave them in the uncategorized group since there isn't a good way to classify them based on their json

Reply:
@aaron-congo re. FCALL - we already have a fallback that if key count == 0, meaning that no keys are passed, we will route the request to a random node. So please add FCALL to be mapped to ThirdArgAfterKeyCount

```
            RouteBy::ThirdArgAfterKeyCount => {
                let key_count = r
                    .arg_idx(2)
                    .and_then(|x| std::str::from_utf8(x).ok())
                    .and_then(|x| x.parse::<u64>().ok())?;
                if key_count == 0 {
                    Some(RoutingInfo::SingleNode(SingleNodeRoutingInfo::Random))
                } else {
                    r.arg_idx(3).map(|key| RoutingInfo::for_key(cmd, key))
                }
            }

Reply:
since cluster_routing can handle optional keys if there is a numkeys argument before the optional key, I've updated the script to categorize these cases and have updated FCALL to map to ThirdArgAfterKeyCount, this PR should be good to merge now once it has approvals